### DISCUSSION TITLE
myBackgroundMessageHandler not called

### DISCUSSION THREAD

**Discussion by umagloire99** [firebase/flutterfire#3939] [Q&A]
Hello, Iam developing an app for pushing notifications from firebase cloud notifications, when the app in background or terminated and receive notification i need to save the data of this notification local in sqlite , without click on the notification or re-open the app gain. the background message handler not called or read the message when the app in background. 
help please!
-`firebase_messaging: ^7.0.0`

**Comment by dackers86** [comment]
Hi @umagloire99.

Please see https://twitter.com/mikediarmid/status/1319298281290203139

**Reply by umagloire99** [reply]
Thanks @dackers86 hope that the new release go solve my problem
